### PR TITLE
Changed styling to new mockup, moved components to seprate files.

### PR DESCRIPTION
**Pull request by hubert-szczepanski**

### CODE REVIEW

File: src/components/KudoBoard.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                     </v-list>
+                </v-card>
+            </v-flex>
+        </v-layout>
+    </v-content>
+</template>
+
+<script>
+import axios from 'axios';
+import Toolbar from './Toolbar'
```
Comment (Bartosz-Rockwool) [inline src/components/KudoBoard.vue]: is it necessary?

File: src/components/KudoBoard.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import KudoForm from './KudoForm'
```
Comment (Bartosz-Rockwool) [inline src/components/KudoBoard.vue]: is it necessary?

File: src/components/KudoBoard.vue [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+import KudoBoard from './KudoBoard'
```
Comment (Bartosz-Rockwool) [inline src/components/KudoBoard.vue]: is it necessary?

File: src/components/KudoBoard.vue (hunk 1 above)
Comment (hubert-szczepanski) [reply src/components/KudoBoard.vue]: It is necessary to first import component if you want to use it :) The same as library in .NET

File: src/components/KudoBoard.vue (hunk 2 above)
Comment (hubert-szczepanski) [reply src/components/KudoBoard.vue]: It is necessary to first import component if you want to use it :) The same as library in .NET

File: src/components/KudoBoard.vue (hunk 3 above)
Comment (hubert-szczepanski) [reply src/components/KudoBoard.vue]: It is necessary to first import component if you want to use it :) The same as library in .NET

**Pull request merged**